### ISSUE TITLE
ChainedStateBuilder fails with only one state

### ISSUE THREAD

**Issue by fcollman** [seung-lab/NeuroglancerAnnotationUI#19]
I think that an edge case is missed that using a single statebuilder for the ChainedStateBuilder doesn't work

**Comment by fcollman** [comment]
Actually turns out the issue is with selecting an individual segID but passing no data.

```from nglui.statebuilder import from_client, StateBuilder
from caveclient import CAVEclient
max_input_neuron=864691135303399847
img_layer ,seg_layer = from_client(client)
seg_layer.add_selection_map(fixed_ids=864691135303399847)
sb = StateBuilder([img_layer, seg_layer])
sb.render_state(return_as='html')```

**Comment by ceesem** [comment]
Ah, numpy integer "array" handling strikes again. Fixed by [5ef718f](https://github.com/seung-lab/NeuroglancerAnnotationUI/commit/5ef718f0bb7dd82be0a761a5b192eeb1c9aa0baa)
